.github/workflows/twisted_trunk.yml

```diff
@@ -55,11 +65,24 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Patch dependencies
-        run: .ci/patch_for_twisted_trunk.sh
+        # Note: The script below runs poetry against a virtualenv in /src/.venv/
+        #       ...but the sytest-synapse container expects it to be in /venv/
+        #       So we temporarily symlink it, run poetry, then remove the link.
+        #       Tidying up is important because `/src` is a mounted volume.
```

Comment:
I'm not really following why that matters? What cares about things being in `/src` ?

Reply:
I must defer to a higher power here: @squahtx

Reply:
To be fair it probably doesn't matter, since `/src` is a volume mount of `${{ github.workspace }}` and I believe nothing bad happens if you scribble all over it.

(Later on, `scripts/synapse_sytest.sh` in sytest will make a copy of `/src`, `rm -rf .venv` and re-do the symlink. Nothing will break there if a stray virtualenv is left in.)

Reply:
ok, then I suggest we remove the `rm` and update the comment. Have made a suggestion below.